### DISCUSSION TITLE
new Anime Matrix modes

### DISCUSSION THREAD

**Discussion by AlbinoPriest** [seerge/g-helper#1711] [Ideas]
am i the only one who wants a mode like the one in armory crate where you can edit and make ur own animations and gifs. or at the very least import .webmatrix files into the AnimeMatrix page

**Comment by seerge** [comment]
@babbenmb hello, I was answering this already. You can compose your own gif, containing everything that you want and set it in the app :) I won't be turning G-Helper into Photoshop, as whole purpose of the app is to be minimalistic and lightweight.

As for .webmatrix format - it's proprietary format from Asus :)

**Reply by AlbinoPriest** [reply]
ah ok, but would it be possible to get some kind of overlay. that would indicate where the gif or picture would go on the screen. ion know if that makes any sense. because right now its kind hard to line up, especially in the diagonal mode.
